CSharpMath/Atoms/Factories/MathAtoms.cs

```diff
@@ -156,81 +156,81 @@ public static class MathAtoms {
+         { "sin", Operator("sin", false, true) },
+         { "arcsin", Operator("arcsin", false, true) },
+         { "sinh", Operator("sinh", false, true) },
+         { "arsinh", Operator("arsinh", false, true) }, //not in iosMath
+         { "cos", Operator("cos", false, true) },
+         { "arccos", Operator("arccos", false, true) },
+         { "cosh", Operator("cosh", false, true) },
+         { "arcosh", Operator("arcosh", false, true) }, //not in iosMath
+         { "tan", Operator("tan", false, true) },
+         { "arctan", Operator("arctan", false, true) },
+         { "tanh", Operator("tanh", false, true) },
          { "artanh", Operator("artanh", false, true) },  //not in iosMath
-         { "cot", Operator("cot" , false, true) },
+         { "cot", Operator("cot", false, true) },
          { "arccot", Operator("arccot", false, true) },  //not in iosMath
-         { "coth", Operator("coth" , false, true) },
+         { "coth", Operator("coth", false, true) },
          { "arcoth", Operator("arcoth", false, true) },  //not in iosMath
-         { "sec", Operator("sec" , false, true) },
+         { "sec", Operator("sec", false, true) },
          { "arcsec", Operator("arcsec", false, true) },  //not in iosMath
          { "sech", Operator("sech", false, true) },  //not in iosMath
          { "arsech", Operator("arsech", false, true) },  //not in iosMath
-         { "csc", Operator("csc" , false, true) },
+         { "csc", Operator("csc", false, true) },
          { "arccsc", Operator("arccsc", false, true) },  //not in iosMath
          { "csch", Operator("csch", false, true) },  //not in iosMath
          { "arcsch", Operator("arcsch", false, true) },  //not in iosMath
-         { "arg", Operator("arg" , false, true) },
-         { "ker", Operator("ker" , false, true) },
-         { "dim", Operator("dim" , false, true) },
-         { "hom", Operator("hom" , false, true) },
-         { "exp", Operator("exp" , false, true) },
-         { "deg", Operator("deg" , false, true) },
+         { "arg", Operator("arg", false, true) },
+         { "ker", Operator("ker", false, true) },
+         { "dim", Operator("dim", false, true) },
+         { "hom", Operator("hom", false, true) },
+         { "exp", Operator("exp", false, true) },
+         { "deg", Operator("deg", false, true) },
          
          // Limit operators
-         { "lim", Operator("lim" , null) },
-         { "limsup", Operator("lim sup" , null) },
-         { "liminf", Operator("lim inf" , null) },
-         { "max", Operator("max" , null) },
-         { "min", Operator("min" , null) },
-         { "sup", Operator("sup" , null) },
-         { "inf", Operator("inf" , null) },
-         { "det", Operator("det" , null) },
-         { "Pr", Operator("Pr" , null) },
-         { "gcd", Operator("gcd" , null) },
+         { "lim", Operator("lim", null) },
+         { "limsup", Operator("limsup", null) },
+         { "liminf", Operator("liminf" , null) },

```

Comment:
Leave these two as-is. The previous code matches LaTeX behaviour so we will not deviate here. Moreover, having a space between the two words is the standard form in mathematics: https://en.wikipedia.org/wiki/Limit_superior_and_limit_inferior

Reply:
Unresolved